### PR TITLE
Create LICENSE

### LINKED ISSUE #23: Add: License and Domain to show the Project, and Fill about section
### 💡 Describe the solution you'd like

WE need to add license of project copyrights and We can launch our project to showcase(using any Domain(optional .org)).

### 📹 Screenshots or Video or GIF

---

**Note:**
* Do mention if you're working on this issue under ` LGM SOC 2021 `.
* If you want to work on any issue, you should check if it has already been assigned to anyone.

### PR DESCRIPTION
**Pull request by thegautamkumarjaiswal**
Added the GNU General Public License.

## Related Issue

Fixes #23 

## Checklist:

- [x] Have you checked there aren't other open [Pull Requests](https://github.com/Feminine-Divine/Woodog/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video or GIF:

---

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
+    Copyright (C) <year>  <name of author>
```
Comment (hariramjp777) [inline LICENSE:635]: `<year>` will be the current year (2021)
`<name of author>` will be 'Feminine-Divine'

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,674 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    <program>  Copyright (C) <year>  <name of author>
```
Comment (hariramjp777) [inline LICENSE:655]: Same here too. 
`<program>` will be 'Woodog'

Please review this @khushishikhu, @Aaishpra

**Comment by thegautamkumarjaiswal** [comment]
Thank you @hariramjp777 for a review, I correct this.

File: LICENSE [hunk 3]
```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    <one line to give the program's name and a brief idea of what it does.>
-    Copyright (C) <year>  <name of author>
+    <Woodog is an initiative to provide help to increasing stray animals by providing them food and refuge by connecting them with fellow helpful humans.>
+    Copyright (C) <2021>  <Feminine-Divine>
```
Comment (hariramjp777) [inline LICENSE:635]: Hi @thegautamkumarjaiswal, Just 2021 is enough, It doesn't need to be enclosed with angular brackets. Same for all, 
Please update this too. :+1:

**Review by hariramjp777 — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 3 above)
Comment (thegautamkumarjaiswal) [reply LICENSE]: Ok, I do.

**Comment by thegautamkumarjaiswal** [comment]
@khushishikhu @Aaishpra please review.

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,675 @@
...
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    Woodog is an initiative to provide help to increasing stray animals by providing them food and refuge by connecting them with fellow helpful humans.
+    Copyright (C) 2021, Feminine-Divine
```
Comment (hariramjp777) [inline LICENSE:635]: :+1:

**Review by hariramjp777 — APPROVED** [review]
Thanks @thegautamkumarjaiswal

File: LICENSE (hunk 4 above)
Comment (thegautamkumarjaiswal) [reply LICENSE:635]: Thank you for your adaptable reviews.

**Review by khushishikhu — APPROVED** [review]
LGTM!!

**Pull request merged**